apps/web/app/[locale]/page-component.tsx

```diff
@@ -84,25 +75,12 @@ function MainPage() {
 					<TeamInvitations />
 				</MainHeader>
 
-				<div className={`z-50 bg-white dark:bg-[#191A20] pt-5 ${view == IssuesView.TABLE ? 'pb-7' : ''}`}>
+				{/* <div className={`z-50 bg-white dark:bg-[#191A20] `}> */}
 					<Container fullWidth={fullWidth}>
-						{isTeamMember ? <TaskTimerSection isTrackingEnabled={isTrackingEnabled} /> : null}
-						{view === IssuesView.CARDS && isTeamMember ? (
-							<UserTeamCardHeader />
-						) : view === IssuesView.BLOCKS ? (
-							<UserTeamBlockHeader />
-						) : view === IssuesView.TABLE ? (
-							<UserTeamTableHeader />
```

Comment:
@Anishali2 if you removed that Table view here, where you added it back? I.e. previously we had that block of icons that allow user to jump to different views, but I don't see where is Table view after your changes?

Reply:
Sir I'm using tenstack table header now which is more accurate in assessability and responsiveness 

![image](https://github.com/ever-co/ever-teams/assets/105014311/2bcac191-a395-4055-8b84-77241a1884f5)